### DISCUSSION TITLE
Does this support `appDir` for Next 13?

### DISCUSSION THREAD

**Discussion by mikehuebner** [tamagui/tamagui#409] [Q&A]
Just curious if I'm missing something for this or it isn't documented yet. Tried to get it running a few days back and had no luck.

**Comment by pontusab** [comment]
@natew sorry for pinging you, but we tried to get Tamagui to work in our new admin app and we would really want to use `appDir` but getting some errors, is Tamagui supported with Next 13 in ´appDir´?

**Reply by mikehuebner** [reply]
I gave up on trying to get it working, starting looking into what I could do to help support the `withTamagui` plugin. However, I don't know enough about how NextJS plugins work at the moment to contribute much. Was debating using a different framework entirely because of it.

**Comment by izakfilmalter** [comment]
Keen on this as well. Want to move into the future.
